server/public/tachyons.css

```diff
@@ -0,0 +1,7772 @@
+/*! TACHYONS v4.11.2 | http://tachyons.io */
```

Comment:
Can we add the minified version instead? Considering the size of the thing it actually will make a difference in this case: https://raw.githubusercontent.com/tachyons-css/tachyons/master/css/tachyons.min.css